cloudmock/aws/mockec2/instances.go

```diff
@@ -49,3 +49,8 @@ func (m *MockEC2) DescribeInstancesPages(request *ec2.DescribeInstancesInput, ca
 func (m *MockEC2) DescribeInstancesPagesWithContext(aws.Context, *ec2.DescribeInstancesInput, func(*ec2.DescribeInstancesOutput, bool) bool, ...request.Option) error {
 	panic("Not implemented")
 }
+
+func (m *MockEC2) DescribeInstanceTypes(*ec2.DescribeInstanceTypesInput) (*ec2.DescribeInstanceTypesOutput, error) {
+	klog.Warningf("MockEc2::DescribeInstanceTypes is stub-implemented")
```

Comment:
If it helps, we can try to narrow the instance types we support in testing, or just support fake instance types.

Reply:
This function isn't actually used, it just needs to be defined in order to satisfy the mock interface. The actual stub we use in integration testing is in `upup/pkg/fi/cloudup/awsup/mock_aws_cloud.go` where I defined a reasonable mock behavior.